### ISSUE TITLE
Regressietest 14.17.0 error NetworkModelPBL

### ISSUE THREAD

**Issue by jipclaassens** [ObjectVision/GeoDMS#707]
Conclusion: 
arc_length of geometry with null gave in 14.16.0 distance null. But in 14.17.0 it gives value 0.

------------------

------------------

`/RegressieTest/t405_2_NetworkModel_PBL_indicator_results_test/result`

Gives the following error:
`Link_Node2_rel: out of range or undefined values not allowed`

**In 14.16.0 it does work.**

-------------

Results for item:
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/At_07h00m00s/CreateODs/StaticNet_subset/F2`

**In 14.16.0:**
![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/fcc6f4c3-bab6-4bbc-8e76-54a8f0e96673)

**In 14.17.0:**
![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/e8d07036-719b-400a-b0a4-7b498a406768)

---------------

The source item is the same in both versions.
So it comes from a subset operation:
![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/537991ba-f663-4f21-931b-cf9340affb75)

```
unit<uint32> StaticNet_subset_debug := Static_net
{
	attribute<bool>   IsSub_from := FromTime >= inTime ;
	attribute<bool>   IsSub_to   := ToTime   < (inTime + (convert(ModelParameters/MaxOVTime, s_f)[Time] * 2));
	attribute<bool>   IsSub_dur  := duration <= convert(ModelParameters/MaxOVTime , s_f) * 2f;
}

unit<uint32> StaticNet_subset := select_with_attr_by_cond(SourceData/Infrastructuur/GTFS/StaticNets/Static_net, StaticNet_subset_debug/IsSub_from && StaticNet_subset_debug/IsSub_to && StaticNet_subset_debug/IsSub_dur);
```

-------------------
14.16.0
IsSub_from = 48,508,143 true
IsSub_to = 14,588,823 true
IsSub_dur = 46,713,893 true

14.17.0
IsSub_from = 48,508,143 true
IsSub_to = 16,275,820 true
IsSub_dur = 52,010,229 true

---------

Compare `/SourceData/Infrastructuur/GTFS/StaticNets/Static_net/Duration`

14.16.0

![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/3cd1c9c2-a480-4a8e-8dbb-41e700f93a06)

14.17.0
![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/950fd72c-1898-4170-a339-a0d14064f765)

------

Difference is already present in: 
`/SourceData/Infrastructuur/GTFS/StaticNets/allLinks/Duration`

And in:
/SourceData/Infrastructuur/GTFS/StaticNets/CreateTransfersSet/TransfersByStart/Duration

Difference is here:
/SourceData/Infrastructuur/GTFS/StaticNets/CreateTransfersSet/TransferPaths/minTransferTime

--------

TL:DR:

arc_length of geometry with null gave in 14.16.0 distance null. But in 14.17.0 it gives value 0.

![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/76a04c1f-b34a-4565-8095-7a9cb60896f8)

![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/407fd101-11af-4c50-964b-998f83bd64bf)

---------

**Comment by jipclaassens** [comment]
resultaat van #701

**Comment by MaartenHilferink** [comment]
Fixed in 14.17.2: arc_length now results in <null>. For obtaining the length of multi linestrings, now use mls_length.
